server/src/main/scala/com/cloudera/livy/server/batch/BatchSession.scala

```diff
@@ -38,6 +38,10 @@ class BatchSession(
     val conf = prepareConf(request.conf, request.jars, request.files, request.archives,
       request.pyFiles)
     require(request.file != null, "File is required.")
+    require(!request.pyspark_python.exists(s => s != "python2" && s != "python3"),
```

Comment:
I think in a lot of environment, Python 2 is simply `python` (eg. CentOS) - could this support that?